### DISCUSSION TITLE
A way for hiding or killing egui window without exiting the whole process.

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [emilk/egui#1393] [Q&A] [answered]
#### So I have this tray menu and I'd love to kill the Egui window completely when someone clicks the `Hide` button.
![Screenshot 2022-03-21 143350](https://user-images.githubusercontent.com/72823042/159261839-531937a5-a2fa-4833-86fa-5eb5224ad9a6.png)

### Is this functionality available?

```rust
run_native(Box::new(app), win_options);
```

**Comment by emilk** [comment, marked as answer]
No, not currently. This is beeing discussed in https://github.com/emilk/egui/issues/1223

In the meanwhile you could consider making your app 0x0 pixels big, and/or making it transparent.
